shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/presentation/modules/surveydetail/SurveyDetailViewModel.kt

```diff
@@ -86,4 +139,42 @@ class SurveyDetailViewModel(
             )
         }
     }
+
+    private fun setNextQuestionState() {
+        val questionSize = viewState.value.surveyDetail?.questions?.size ?: 0
+        val currentState = questionViewState.value
+        var nextQuestionIndex = currentState.currentQuestionIndex + 1
+        val isFinalQuestion = nextQuestionIndex >= (questionSize - 1)
+        val newState = SurveyQuestionViewState(
+            isFinalQuestion,
+            currentState.isLoading,
+            nextQuestionIndex
+        )
```

Comment:
Same here, you should write the parameters name out fully for better readability 🙏
```suggestion
        val newState = SurveyQuestionViewState(
            isShowingSubmit = isFinalQuestion,
            isLoading = currentState.isLoading,
            currentQuestionIndex = nextQuestionIndex
        )
```